opencog/nlp/learn/README

```diff
@@ -344,24 +340,22 @@ where type 7 is `CountTruthValue`.
 Mutual Information of Word Pairs
 --------------------------------
 After accumulating a few million word pairs, we're ready to compute
-the mutual entropy between them.  This is done manually, by running
-functions defined in `(opencog nlp learn)`.
-
+the mutual entropy between them.  This is done by running
 ```
-	(use-modules (opencog) (opencog persist) (opencog persist-sql))
-	(use-modules (opencog matrix))
-	(use-modules (opencog nlp) (opencog nlp learn))
-	(sql-open "postgres:///en_pairs?user=ubuntu&password=asdf")
-	(define ala (make-any-link-api))
-	(define asa (add-pair-stars ala))
-	(batch-pairs asa)
-	(print-matrix-summary-report asa)
+guile -l compute-mi-launch.scm  -- --db ??-pairs --user your_user --password asdf
```

Comment:
Careful adding premature automation.  Some things are better done by hand. Scripting too much too soon is dangerous.  This part might be OK, but at this stage, I'm not sure its robust enough to be fully automated.

Reply:
I changed the guile file to do something easy like just running the commands you provided :)